### PR TITLE
[Enhancement] Create an option to disable Pokemon battle cries

### PR DESCRIPTION
**Pull request by EmberCM**
## What are the changes?
Allows users to disable Pokemon battle cries

## Why am I doing these changes?
[Discord thread](https://discord.com/channels/1125469663833370665/1248907216086368256)

## What did change?
Create an on/off option in Audio category for Pokemon Battle Cries

### Screenshots/Videos
ss

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

tests

### CODE REVIEW

File: src/locales/zh_CN/settings.ts [hunk 1]
```diff
@@ -64,6 +64,7 @@ export const settings: SimpleTranslationEntries = {
   "bgmVolume": "音乐",
   "seVolume": "音效",
   "musicPreference": "音乐偏好",
+  "pokemonCries": "Pokemon Battle Cries",
```
Comment (40chyan) [inline src/locales/zh_CN/settings.ts:67]:
```suggestion
  "pokemonCries": "宝可梦叫声",
```
zh_cn translation

File: src/locales/zh_TW/settings.ts [hunk 2]
```diff
@@ -64,6 +64,7 @@ export const settings: SimpleTranslationEntries = {
   "bgmVolume": "音樂",
   "seVolume": "音效",
   "musicPreference": "音樂偏好",
+  "pokemonCries": "Pokemon Battle Cries",
```
Comment (40chyan) [inline src/locales/zh_TW/settings.ts:67]:
```suggestion
  "pokemonCries": "寶可夢叫聲",
```
zh_tw

File: src/locales/ko/settings.ts [hunk 3]
```diff
@@ -64,6 +64,7 @@ export const settings: SimpleTranslationEntries = {
   "bgmVolume": "BGM 볼륨",
   "seVolume": "SE 볼륨",
   "musicPreference": "음악 설정",
+  "pokemonCries": "Pokemon Battle Cries",
```
Comment (EnochG1) [inline src/locales/ko/settings.ts:67]:
```suggestion
  "pokemonCries": "포켓몬 울음소리",
```

File: src/locales/de/settings.ts [hunk 4]
```diff
@@ -64,6 +64,7 @@ export const settings: SimpleTranslationEntries = {
   "bgmVolume": "Hintergrundmusik",
   "seVolume": "Spezialeffekte",
   "musicPreference": "Musik Präferenz",
+  "pokemonCries": "Pokemon Battle Cries",
```
Comment (CodeTappert) [inline src/locales/de/settings.ts:67]:
```suggestion
  "pokemonCries": "Pokémon Rufe",
```

File: src/locales/en/settings.ts [hunk 5]
```diff
@@ -64,6 +64,7 @@ export const settings: SimpleTranslationEntries = {
   "bgmVolume": "BGM Volume",
   "seVolume": "SE Volume",
   "musicPreference": "Music Preference",
+  "pokemonCries": "Pokemon Battle Cries",
```
Comment (CodeTappert) [inline src/locales/en/settings.ts:67]:
```suggestion
  "pokemonCries": "Pokémon Battle Cries",
```

File: src/locales/fr/settings.ts [hunk 6]
```diff
@@ -64,6 +64,7 @@ export const settings: SimpleTranslationEntries = {
   "bgmVolume": "Vol. musique",
   "seVolume": "Vol. effets",
   "musicPreference": "Préf. musicale",
+  "pokemonCries": "Pokemon Battle Cries",
```
Comment (Adri1) [inline src/locales/fr/settings.ts:67]:
```suggestion
  "pokemonCries": "Cris Pokémon en combat",
```

**Review by Adri1 — COMMENTED** [review]
Added French

File: src/locales/pt_BR/settings.ts [hunk 7]
```diff
@@ -64,6 +64,7 @@ export const settings: SimpleTranslationEntries = {
   "bgmVolume": "Volume de BGM",
   "seVolume": "Volume de SE",
   "musicPreference": "Preferência de Música",
+  "pokemonCries": "Pokemon Battle Cries",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/settings.ts:67]:
```suggestion
  "pokemonCries": "Sons de Batalha Pokémon",
```

**Comment by torranx** [comment]
cant you just `return` early inside the `cry()` method if the setting is disabled?

**Comment by EmberCM** [comment]
> cant you just `return` early inside the `cry()` method if the setting is disabled?

No because this is only for cries during battle. That would disable all cries such as when you check the summary of your Pokemon or hover over it in the starter selection.

**Comment by NicusPulcis** [comment]
@EmberCM added italian translation

**Review by EnochG1 — APPROVED** [review]

**Review by 40chyan — APPROVED** [review]

**Comment by frutescens** [comment]
Hi, thank you for your work. Pokemon cries and battle sounds will have their own volume setting in the next release, so this PR will be closed.

**Pull request closed without merging**